pkg/k8s/watchers/watcher.go

```diff
@@ -160,9 +160,10 @@ type svcManager interface {
 	DeleteService(frontend loadbalancer.L3n4Addr) (bool, error)
 	GetDeepCopyServiceByFrontend(frontend loadbalancer.L3n4Addr) (*loadbalancer.SVC, bool)
 	UpsertService(*loadbalancer.SVC) (bool, loadbalancer.ID, error)
-	RegisterL7LBService(serviceName loadbalancer.ServiceName, resourceName service.L7LBResourceName, proxyPort uint16) error
-	RegisterL7LBServiceBackendSync(serviceName loadbalancer.ServiceName, resourceName service.L7LBResourceName, ports []string) error
-	RemoveL7LBService(serviceName loadbalancer.ServiceName, resourceName service.L7LBResourceName) error
+	RegisterL7LBServiceForward(serviceName loadbalancer.ServiceName, resourceName service.L7LBResourceName, proxyPort uint16) error
```

Comment:
Small nit on naming, for the sake of consistency I'd c all this `RegisterL7LBServiceRedirect` instead.

Reply:
agree: renamed all occurrences from `forward` to `redirect`